README.md

```diff
@@ -1,70 +1,71 @@
-# [BitcoinCash.js](https://bitcoincashjs.github.io/): The simple, safe, and powerful JavaScript library for Bitcoin Cash
+# [BitcoinCashFlow](https://github.com/BitcoinDB/BitcoinCashFlow): The simple, safe, reliable and powerful JavaScript library for Bitcoin Cash
 
-[![NPM Package](https://img.shields.io/npm/v/bitcoincashjs.svg?style=flat-square)](https://www.npmjs.org/package/bitcoincashjs)
-[![Build Status](https://img.shields.io/travis/bitcoincashjs/bitcoincashjs.svg?branch=master&style=flat-square)](https://travis-ci.org/bitcoincashjs/bitcoincashjs)
-[![Coverage Status](https://img.shields.io/coveralls/bitcoincashjs/bitcoincashjs.svg?style=flat-square)](https://coveralls.io/r/bitcoincashjs/bitcoincashjs)
+[![NPM Package](https://img.shields.io/npm/v/bitcoincashflow.svg?style=flat-square)](https://www.npmjs.org/package/bitcoincashflow)
```

Comment:
We will obviously need to configure NPM, Travis and Coveralls for the badges to work.

Reply:
Yup. Maybe we should create issues for this